src/Wrapper/GkmsClientFactory.php

```diff
@@ -4,27 +4,62 @@
 
 namespace Keboola\ObjectEncryptor\Wrapper;
 
+use Google\Auth\HttpHandler\HttpHandlerFactory;
 use Google\Cloud\Kms\V1\KeyManagementServiceClient;
+use GuzzleHttp\BodySummarizer;
+use GuzzleHttp\Client;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Middleware;
 use Keboola\ObjectEncryptor\EncryptorOptions;
 use Keboola\ObjectEncryptor\Exception\ApplicationException;
 use Throwable;
 
 class GkmsClientFactory
 {
+    public const TIMEOUT = 120;
+    public const CONNECT_TIMEOUT = 10;
```

Comment:
Proc tu mame zase jiny hodnoty tech timeoutu, nez jsme se bavili v pondeli spolu? V messengeru jsme to nastavili na `TIMEOUT: 60`, `CONNECT_TIMEOUT` se nenastavoval vubec. Cucham ze messenger budu jeste predelavat, co? :slightly_smiling_face: